src/Bindings/ManualBindings.cpp

```diff
@@ -4546,6 +4547,14 @@ void cManualBindings::Bind(lua_State * tolua_S)
 			tolua_function(tolua_S, "Intersect",            tolua_cBoundingBox_Intersect);
 		tolua_endmodule(tolua_S);
 
+		tolua_beginmodule(tolua_S, "cChunkDef");
+			tolua_constant(tolua_S, "Width",              cChunkDef::Width);
+			tolua_constant(tolua_S, "LowerLimit",         cChunkDef::LowerLimit);
+			tolua_constant(tolua_S, "UpperLimit",         cChunkDef::UpperLimit);
+			tolua_constant(tolua_S, "VerticalBlockCount", cChunkDef::VerticalBlockCount);
```

Comment:
I think this should be exported as `cChunkDef.Height`; once we clean up the C++ code so that it doesn't use `Height` as the upper limit, we can change the C++ counterpart to `Height`, too.